src/main/java/io/swagger/codegen/v3/generators/java/MicronautCodegen.java

```diff
@@ -155,17 +169,18 @@ public void processOpts() {
             skipSupportFiles = Boolean.valueOf(additionalProperties.get(SKIP_SUPPORT_FILES).toString());
         }
 
-        if (useBeanValidation) {
-            writePropertyBack(USE_BEANVALIDATION, useBeanValidation);
-        }
+        writePropertyBack(USE_BEANVALIDATION, useBeanValidation);
```

Comment:
Looks like handlebars only understands booleans, but not booleans represented as strings. So, we have to put boolean to `additionalProperties` instead of its string representation.